src/TConsole.cpp

```diff
@@ -959,13 +927,6 @@ void TConsole::changeColors()
         if (mpCommandLine) {
             auto styleSheet = mpCommandLine->styleSheet();
             mpCommandLine->setStyleSheet(QString());
-            QPalette pal;
-            pal.setColor(QPalette::Text, mpHost->mCommandLineFgColor); //QColor(0,0,192));
-            pal.setColor(QPalette::Highlight, QColor(0, 0, 192));
-            pal.setColor(QPalette::HighlightedText, QColor(Qt::white));
-            pal.setColor(QPalette::Base, mpHost->mCommandLineBgColor); //QColor(255,255,225));
-            mpCommandLine->setPalette(pal);
-            mpCommandLine->mRegularPalette = pal;
```

Comment:
okay after testing. It is this part which messes up with changing command line background and textcolor through settings.

Reply:
:man_facepalming:

Ah, yeah I might have gone too far there...

... let me adjust things a bit. :hammer:

Reply:
Done in https://github.com/Mudlet/Mudlet/pull/4650/commits/bb2da9d4943c5eee6086e1045815127005697a7b .

Reply:
Seems to be fixed. :+1: